app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -653,14 +653,14 @@ private String getIVText() {
                 returnVal += "<br>"; //breakline
                 //for following stage evolution (example, dratini - dragonair - dragonite)
                 //if the current evolution has another evolution calculate its range and break
-                for (Pokemon nextEvo : evolution.evolutions) {
+                if(evolution.evolutions.size()>0) {
```

Comment:
You can still write new code with proper style. Please fix it.

Also, does the coding style need space on either side of ">" ?